pom.xml

```diff
@@ -106,29 +102,22 @@
   <!-- Overriding versions coming from RMB -->
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>io.vertx</groupId>
-        <artifactId>vertx-unit</artifactId>
-        <version>${vertx.version}</version>
-      </dependency>
-
-      <dependency>
-        <groupId>io.vertx</groupId>
-        <artifactId>vertx-dropwizard-metrics</artifactId>
-        <version>${vertx.version}</version>
-      </dependency>
-
       <dependency>
         <groupId>com.fasterxml.jackson</groupId>
         <artifactId>jackson-bom</artifactId>
-        <version>${jackson.version}</version>
+        <version>2.10.1</version>
         <scope>import</scope>
         <type>pom</type>
       </dependency>
```

Comment:
Please completely remove jackson-bom.

This is already provided by vertx-stack-depchain in a more recent version (2.10.2) that should be been downgraded.

Reply:
> This is already provided by vertx-stack-depchain